app/segments/(team)/invitelist.js

```diff
@@ -0,0 +1,142 @@
+import { useAuthContext } from "~/context/Auth";
+import { addToInvitelist } from "~/dbOperations/addToInvitelist";
+import { useInvitelist } from "~/dbOperations/hooks/useInviteList";
+import { invalidateMultipleKeys } from "~/dbOperations/invalidateMultipleKeys";
+import { removeInvitelist } from "~/dbOperations/removeInvitelist";
+
+export function Invitelist() {
+  const {
+    data: invitelist,
+    error: inviteError,
+    isLoading: inviteIsLoading,
+  } = useInvitelist();
+
+  const queryClient = useQueryClient();
+
+  const { currentTeamId } = useAuthContext();
+
+  const invalidateKeys = [["invitelist"]];
+  // const queryClient = useQueryClient();
+
+  const [currentEmailInput, setCurrentEmailInput] = useState("");
+  const [currentEmailValid, setCurrentEmailValid] = useState(false);
+
+  const onInvite = async () => {
+    const invitedEmail = Object.values(invitelist).map(
+      (invite) => invite["email"],
+    );
+    if (invitedEmail.includes(currentEmailInput)) {
+    }
+    await addToInvitelist(currentTeamId, currentEmailInput);
+    setCurrentEmailInput("");
+    await invalidateMultipleKeys(queryClient, invalidateKeys);
+  };
+
+  if (inviteError) {
+    return <ErrorComponent error={getErrorString(inviteError)} />;
+  }
+
+  if (inviteIsLoading) {
+    return <Loading />;
+  }
+
+  const emailRegex = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w\w+)+$/;
+
+  return (
+    <KeyboardAvoidingView
+      style={{
+        flex: 1,
+      }}
+      behavior={Platform.OS === "ios" ? "padding" : "height"}
+    >
+      <ScrollView
+        refreshControl={<RefreshInvalidate invalidateKeys={invalidateKeys} />}
+      >
+        <List.Section
+          style={{
+            margin: 5,
+            borderRadius: 5,
+          }}
+        >
+          {Object.keys(invitelist).map((inviteId) => {
+            return (
+              <List.Item
+                title={invitelist[inviteId].email}
+                key={inviteId}
+                right={() => (
+                  <View
+                    style={{
+                      flexDirection: "row",
+                      alignItems: "center",
+                      paddingLeft: 10,
+                    }}
+                  >
+                    <Button
+                      onPress={async () => {
+                        await removeInvitelist(currentTeamId, inviteId);
+                        await invalidateMultipleKeys(
+                          queryClient,
+                          invalidateKeys,
+                        );
+                      }}
+                      textColor={themeColors.accent}
+                    >
+                      Remove
+                    </Button>
+                  </View>
+                )}
+              />
+            );
+          })}
+        </List.Section>
+      </ScrollView>
+      <View
+        style={{
+          flexDirection: "row",
+          padding: 5,
+        }}
+      >
+        <TextInput
```

Comment:
There is a weird UI bug on android: 
![image](https://github.com/user-attachments/assets/a8c13708-9dc4-4912-b174-9782a25e218b)

Reply:
It fixes when you start typing but on the first click it looks like this

Reply:
fixed!